### DISCUSSION TITLE
Any Expert Advice for Two Admin Panels Backpack, and Old Admin

### DISCUSSION THREAD

**Discussion by ShariqAyaz** [Laravel-Backpack/community-forum#1034] [General]
Before Backpack admin, I already had custom admin panel and it has to customised application that seems very hard to do it on backpack admin and all end-users are trained on it. so i keep it both, make different routes for both admins. Luckily on both same user/password works which is fine. But it ask login id twice one for Backpack and one for Old one.

I need to put values in session or do something that will not ask if any of admin panel logined.

Is there anyway?

that is old admin post request:
```
    public function store(LoginRequest $request): RedirectResponse
    {
        $request->authenticate();

        $request->session()->regenerate();

        return redirect()->intended(RouteServiceProvider::HOME);
    }
```

Thank you

**Comment by pxpm** [comment]
Hello @ShariqAyaz 

By default Laravel uses the `web` guard, and Backpack uses the `backpack`. 

So most likely you can configure Backpack to use the same `web` guard as Laravel, and it will just work ? https://github.com/Laravel-Backpack/CRUD/blob/01270cd35ea45e0c42b9ca65e73e20e1dce41fa1/src/config/backpack/base.php#L136

Unless they are in separate domains. In that case you need to build some custom token authentication, as the authentication cookies are not shared across domains. 

Hope this helps to figure out the answer. 🙏 

Cheers
